src/context/Theme.jsx

```diff
@@ -0,0 +1,14 @@
+import {createContext} from 'react';
+import {useState} from 'react';
+
+export const ThemeContext = createContext()
+
+export const ThemeProvider = ({children}) => {
+    const [activeUser, setActiveUser] = useState(null)
+
+    return (
+        <ThemeContext.Provider value={{activeUser, setActiveUser}}>
+            {children}
+        </ThemeContext.Provider>
+    )
```

Comment:
A great use case for context, as it's used throughout the app and prevents the need for a lot of prop drilling